File: scripts/py-compile-suite/inputs/10_logical_ops.m
% Logical operators
a = 5;
b = 10;
if a > 0 && b > 0
    disp('both positive');
end
if a < 0 || b < 0
    disp('one negative');
end
if ~(a == b)
    disp('not equal');
end
mask = (1:10) > 5;
selected = mask & (mod(1:10, 2) == 0);

File: scripts/py-compile-suite/outputs/10_logical_ops.py
import numpy as np

# Logical operators
a = 5
b = 10
if a > 0 and b > 0:
    print('both positive')
if a < 0 or b < 0:
    print('one negative')
if not (a == b):
    print('not equal')
mask = np.arange(1, 10 + 1) > 5
selected = mask & (np.mod(np.arange(1, 10 + 1), 2) == 0)
